components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2479,6 +2473,12 @@ where
             ExtraMessageType::MsgHibernateResponse => {
                 self.on_hibernate_response(msg.get_from_peer());
             }
+            ExtraMessageType::MsgRejectRaftLogCausedByMemoryUsage => {
```

Comment:
The name is too limited. It can be caused by any reasons.

Reply:
It's only used by memory limit.

Reply:
Why? For example, if disk is nearly full, I want to make leader not sending any entries, I can also use this extra message type.

Reply:
Why not make them use one method?

Reply:
When reject, to send a heartbeat msg is not a good way.

Reply:
I don't understand, why reusing a message becomes a wrong way?

Reply:
Not a wrong way.
Leader turn to Probe, which will be awaken by the follower heartbeat.